aws/check-cloudwatch-alarm.yaml

```diff
@@ -0,0 +1,87 @@
+---
+# Description
+#   Check the status of a cloudwatch alarm
+#
+# Instructions
+#   1. Add the "aws" subscription to agents that should run this check.
+#   2. More configuration options are available for this plugin; please see the
+#      plugin documentation for more details.
+#
+# Documentation
+#   - Usage: https://github.com/sensu-plugins/sensu-plugins-aws#usage-examples
+#
+# Contributors
+#   The following individuals have contributed to this configuration template:
+#   - Todd Campbell, @nixwiz
+type: CheckConfig
+api_version: core/v2
+metadata:
+  name: check-cloudwatch-alarm
+spec:
+  command: >-
+    check-cloudwatch-alarm
+    --aws_region={{ .annotations.check_cloudwatch_alarm_aws_region | default "us-east-1" }}
+    --state={{ .annotations.check_cloudwatch_alarm_state | default "ALARM" }}
+  runtime_assets:
+    - sensu-plugins/sensu-plugins-aws:0.0.5
+  publish: false
+  subscriptions:
+    - aws
```

Comment:
Same comment as above... but maybe even adding a sub-topic subscription like `aws/cloudwatch/alarm`? For now we only have a few Cloudwatch examples, so this might not be necessary at this time (i.e. `aws/cloudwatch` should suffice); but as we add more examples it may be prudent to adopt a sub-topic style subscription. Thoughts?